assets/src/edit-story/elements/video/icon.js

```diff
@@ -30,7 +29,7 @@ import VisibleImage from '../media/visibleImage';
 
 function VideoLayerContent({
   element: {
-    resource: { poster: defaultPoster, alt = __('Video', 'web-stories'), src },
+    resource: { poster: defaultPoster, alt, src },
```

Comment:
The icon and the layer text both use the `alt` of the resource.

The title gets repeated as text content, but the layer panel isn't read at all by voice over anyways. Wondering if we should come back to this later to make it make more sense to screen readers - like reading 'Earth icon - Earth' 🤔

Reply:
If they're both the same, seems like the icon should be marked as decorative by using `alt=""`. Then only the title is read.